pallets/automation-time/src/lib.rs

```diff
@@ -460,6 +546,25 @@ pub mod pallet {
 			10_000
 		}
 
+		fn run_transfer_task(
+			sender: T::AccountId,
+			recipient: T::AccountId,
+			amount: BalanceOf<T>,
+			task_id: T::Hash,
+		) -> Weight {
+			match <T as Config>::Currency::transfer(
+				&sender,
+				&recipient,
+				amount,
+				ExistenceRequirement::KeepAlive,
+			) {
+				Ok(_number) => Self::deposit_event(Event::SuccesfullyTransferredFunds { task_id }),
+				Err(e) => Self::deposit_event(Event::TransferFailed { task_id, error: e }),
```

Comment:
acceptably pretty error? 
![image](https://user-images.githubusercontent.com/22332240/154152248-64229b51-1016-47f6-b5e6-63a522ebaa7b.png)

my only concern here is: is there anything that's going to be "PII" that's being printed? i don't think so, since blockchain is all public anyway, but just curious

Reply:
well these errors are what the Balance transfer API would return right?